### PR TITLE
Renaming follow-up changes

### PR DESCRIPTION
**Pull request by Wapwolf**
A very important and life-changing submission

## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/etna-team/etna/blob/master/CONTRIBUTING.md)?
- [ ] Did you update the docs? We use Numpy format for all the methods and classes.
- [ ] Did you write any new necessary tests?
- [ ] Did you update the [CHANGELOG](https://github.com/etna-team/etna/blob/master/CHANGELOG.md)?

## Proposed Changes

* Fixed the naming and link to the company after renaming from tinkoff to t-bank

### CODE REVIEW

**Comment by d-a-bunin** [comment]
Hi! What do you think about updating the license? 

https://github.com/etna-team/etna/blob/master/LICENSE#L178

**Comment by Wapwolf** [comment]
tbh me too, this is a question for the IP lawyer. But my guess would be that yes, it should also be updated

**Comment by d-a-bunin** [comment]
I have in mind some other places to update:
- `pyproject.toml`
  - Description field
  - Emails for the initial developers
- `docs/source/conf.py`
  - `copyright`
  - `author`
- `docs/source/index.rst`

**Comment by Wapwolf** [comment]
Fixed

> I also found some other places to update:
> 
> * `pyproject.toml`
>   
>   * Description field
>   * Emails for the initial developers
> * `docs/source/conf.py`
>   
>   * `copyright`
>   * `author`
> * `docs/source/index.rst`

Fixed

File: LICENSE [hunk 1]
```diff
@@ -175,7 +175,7 @@
 
    END OF TERMS AND CONDITIONS
 
-   Copyright 2021 Tinkoff Bank
+   Copyright 2021 T-Bank
```
Comment (d-a-bunin) [inline LICENSE:178]: Wasn't it decided not to do this renaming in LICENSE?

File: docs/source/index.rst [hunk 2]
```diff
@@ -8,7 +8,7 @@ to SOTA neural networks, models combination methods and smart backtesting.
 ETNA is designed to make working with time series simple, productive, and fun.
 
 ETNA is the first python open source framework of
-`Tinkoff.ru <https://www.tinkoff.ru/eng/>`_ Artificial Intelligence Center.
+`Tinkoff.ru <https://www.tbank.ru/>`_ Artificial Intelligence Center.
```
Comment (d-a-bunin) [inline docs/source/index.rst:11]: Probably, Tinkoff.ru here should also be renamed.

**Review by d-a-bunin — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (Wapwolf) [reply LICENSE:178]: Hmm, forgot to pull, sorry. Fixed in newest commit

File: docs/source/index.rst (hunk 2 above)
Comment (Wapwolf) [reply docs/source/index.rst]: Fixed

File: LICENSE (hunk 1 above)
Comment (d-a-bunin) [reply LICENSE:178]: It seems like it wasn't fixed by the newest commit. The most reliable source of a change is a tab "Files changed" in the PR.

Comment (Wapwolf) [reply LICENSE]: Well, now :Files changed" tab does not even show the change in LICENSE. But I guess I fixed my mistake, now it surely has the correct version

**Review by d-a-bunin — APPROVED** [review]

**Pull request merged**